specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json

```diff
@@ -533,6 +566,55 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Consumption/reservationRecommendations": {
+      "get": {
+        "tags": [
+          "ReservationRecommendations"
+        ],
+        "operationId": "reservationRecommendations_List",
```

Comment:
Should be "ReservationRecommendations_List" (upper-case 'r').

Reply:
Sure! Corrected.